### PR TITLE
Improvement/general improvements

### PR DESCRIPTION
**Pull request by TomasKostadinov**
This PR contains

- General Styling improvements

- Image Galleries

- Image Gallery Overview page

### CODE REVIEW

File: site/aktuelles/index.html [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+    <div class="hero hero--image hero--small hero--light"
         style="background: linear-gradient(0deg,#FCC509A3,#FCC509B8),url('/assets/images/aktuelles/titelbild.jpg') bottom right / cover;">
         <div class="container">
             <div class="hero-inner">
                 <div class="hero-card">
                     <h1>Aktuelles </h1>
                     <h3>
                         Hier finden sie die neuesten Informationen </br>
-                        und bleiben so immer up to date.
+                        und bleiben so immer up-to-date.
```
Comment (BlueEnni) [inline site/aktuelles/index.html:11]: In my opinion the hyphen look weird

File: site/assets/styles/styles.css [hunk 2]
```diff
@@ -448,7 +477,7 @@ a {
 }
 
 .blocks.blocks--even {
-    grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```
Comment (BlueEnni) [inline site/assets/styles/styles.css:511]: solves some overlap problems? - can you bring that up tomorrow at our meeting?

File: site/pages.json [hunk 3]
```diff
@@ -566,15 +576,15 @@
...
-          }
+              "name": "mobi-stabi-training",
+              "destination": "trainingsplan",
+              "file": "triathlon/mobi-stabi-training/trainingsplan/index.html",
+              "title": "mobi-stabi-training",
+              "photoUrl": "https://svm-dh.herokuapp.com/assets/images/running1.jpg",
+              "description": "mobi-stabi-training",
+              "children": [
+              ]
+            }
```
Comment (BlueEnni) [inline site/pages.json]: no need for that

File: site/pages.json [hunk 4]
```diff
@@ -586,39 +596,39 @@
...
+              "destination": "trainingsplan",
+              "file": "triathlon/radfahren/trainingsplan/index.html",
+              "title": "radfahren",
+              "photoUrl": "https://svm-dh.herokuapp.com/assets/images/running1.jpg",
+              "description": "radfahren",
+              "children": [
+              ]
+            }
+          ]
+        },
```
Comment (BlueEnni) [inline site/pages.json]: this aswell

File: site/pages.json [hunk 5]
```diff
@@ -586,39 +596,39 @@
...
+          "children": [
+            {
+              "name": "schwimmen",
+              "destination": "trainingsplan",
+              "file": "triathlon/schwimmen/trainingsplan/index.html",
+              "title": "schwimmen",
+              "photoUrl": "https://svm-dh.herokuapp.com/assets/images/running1.jpg",
+              "description": "schwimmen",
+              "children": [
+              ]
```
Comment (BlueEnni) [inline site/pages.json]: this aswell

File: site/template/index.html [hunk 6]
```diff
@@ -53,42 +53,26 @@
...
-                </ul>
-            </div>
-            <div class="footer-item">
-                <ul>
-                    <li><a href="#">Impressum</a></li>
-                    <li><a href="#">Kontakt</a></li>
-                    <li><a href="#">Email</a></li>
-                    <li><a href="#">Telefon</a></li>
+                    <li><a href="/impressum">Impressum</a></li>
+                    <li><a href="/impressum">Datenschutz</a></li>
```
Comment (BlueEnni) [inline site/template/index.html]: we could add the team here too?

**Review by BlueEnni — COMMENTED** [review]
a couple of remarks

File: site/pages.json [hunk 7]
```diff
@@ -546,15 +556,15 @@
...
-            ]
-          }
+              "name": "Laufen",
+              "destination": "trainingsplan",
+              "file": "triathlon/laufen/trainingsplan/index.html",
+              "title": "Laufen",
+              "photoUrl": "https://svm-dh.herokuapp.com/assets/images/running1.jpg",
+              "description": "Laufen",
+              "children": [
+              ]
```
Comment (BlueEnni) [inline site/pages.json]: no need for that - all trainingsplan pages can be removed - in the branch improvements/triathlon_startpage I did remove those already

File: site/template/index.html (hunk 6 above)
Comment (TomasKostadinov) [reply site/template/index.html]: will do

File: site/assets/styles/styles.css (hunk 2 above)
Comment (TomasKostadinov) [reply site/assets/styles/styles.css:511]: just ask I might forget

File: site/aktuelles/index.html (hunk 1 above)
Comment (TomasKostadinov) [reply site/aktuelles/index.html:11]: both are correct but i think this is the more german way

**Review by DominikRuf — APPROVED** [review]
Ok

**Pull request merged**